src/modules/application/containers/TradeWidgetLinks/index.tsx

```diff
@@ -15,12 +17,18 @@ interface MenuItemConfig {
   route: RoutesValues
   label: string
   featureGuard?: string
+  onClick?: () => void
 }
 
 const menuItems: MenuItemConfig[] = [
   { route: Routes.SWAP, label: 'Swap' },
   { route: Routes.LIMIT_ORDER, label: 'Limit' },
-  { route: Routes.ADVANCED_ORDERS, label: 'Advanced', featureGuard: 'advancedOrdersEnabled' },
+  {
+    route: Routes.ADVANCED_ORDERS,
+    label: 'Advanced',
+    featureGuard: 'advancedOrdersEnabled',
+    onClick: clickAdvancedOrdersTabAnalytics,
```

Comment:
You can replace it by adding code to `TwapFormWidget`:

```
useEffect(() => {
    sendEvent({
        category: Category.TWAP,
        action: 'Click Advanced Orders Tab',
    })
}, [])
```

Reply:
Good suggestion, updated 👍 Now the action will be `Open Advanced Orders Tab`